components/zoho_creator/sources/new-or-updated-record/new-or-updated-record.mjs

```diff
@@ -0,0 +1,85 @@
+import common from "../common.mjs";
+import utils from "../../utils.mjs";
+import constants from "../../constants.mjs";
+
+const { zohoCreator } = common.props;
+const { toSingleLineString } = utils;
+
+export default {
+  ...common,
+  key: "zoho_creator-new-or-updated-record",
+  description: toSingleLineString(`
+    Emit new or updated records in a report. The \`Added Time\` and \`Modified Time\` fields must be added as
```

Comment:
Yeah actually I had to add it otherwise an error will raise whenever I send the request with Modified Time criteria

Reply:
Is the "Added Time" field required for this source? I might be doing something wrong, but I am not getting an error when I create a report grouping with only the "Modified Time" field, and remove these lines from the source:
- https://github.com/PipedreamHQ/pipedream/blob/7e3adf5fdba1ca59ce456443f7b3e8b39850c16b/components/zoho_creator/sources/new-or-updated-record/new-or-updated-record.mjs#L53
- https://github.com/PipedreamHQ/pipedream/blob/7e3adf5fdba1ca59ce456443f7b3e8b39850c16b/components/zoho_creator/sources/new-or-updated-record/new-or-updated-record.mjs#L70
- https://github.com/PipedreamHQ/pipedream/blob/7e3adf5fdba1ca59ce456443f7b3e8b39850c16b/components/zoho_creator/sources/new-or-updated-record/new-or-updated-record.mjs#L77

<img width="550" alt="grouping-modified_time" src="https://user-images.githubusercontent.com/19861096/146048687-5aa3142b-4c70-4134-87c4-5c6569aa222c.png">

Reply:
Oh yeah thanks! I've got confused by another error because I wasn't sending the time format in the field. With only Modified Time would be good enough.